src/VisualStudio/Core/Def/Implementation/ProjectSystem/MiscellaneousFilesWorkspace.cs

```diff
@@ -353,7 +353,10 @@ private ProjectInfo CreateProjectInfoForDocument(string filePath)
 
             var languageServices = Services.GetLanguageServices(languageInformation.LanguageName);
             var compilationOptionsOpt = languageServices.GetService<ICompilationFactoryService>()?.GetDefaultCompilationOptions();
-            var parseOptionsOpt = languageServices.GetService<ISyntaxTreeFactoryService>()?.GetDefaultParseOptions();
+
+            // We only have one code file without project file here, so we cannot find out language version of the project  
```

Comment:
Comment seems out of place since that's the fact of life for this entire type. :smile:

Reply:
updated the comment